BaseStation/MainWindow/VideoStreamer/VideoWindow.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuskyRobotics.UI.VideoStreamer
```

Comment:
Should this interface include more streaming related methods such as StartStream?